cola/models/stash.py

```diff
@@ -89,10 +90,12 @@ def do(self):
         status, out, err = git.stash('drop', self.stash_ref)
```

Comment:
For the block below, I've been trying to stick to single-exit.. so the logic flow would be..
```
if ...:
    result = ...
else:
    result = ...
return result
```
That way there's just a single return.  Not always possible, but seems sensible below.

Reply:
Right, I would also like to have your opinion on the return value if there is an error. I chose `''` to make the linter happy (same return type). Is it ok or would you see something else?

Reply:
empty string is fine by me, especially if the linter was complaining about it.  It does make things simpler for callers since they wouldn't have to deal with any `None` vs `str` concerns.